src/Domain/Creators/CylindricalBinaryCompactObject.hpp

```diff
@@ -230,6 +237,8 @@ class CylindricalBinaryCompactObject : public DomainCreator<3> {
       typename RadiusA::type radius_A, typename RadiusB::type radius_B,
       typename OuterRadius::type outer_radius,
       typename InitialRefinement::type initial_refinement,
+      typename AdditionToRadialRefinementLevel::type
+          addition_to_radial_refinement_level,
```

Comment:
it might be worth doing `to_radial` -> `to_outer_radial` to make it clearer where the refinement is done. Up to you :)